src/ffi/pystate.rs

```diff
@@ -21,43 +25,45 @@ extern "C" {
     #[cfg_attr(docsrs, doc(cfg(not(PyPy))))]
     pub fn PyInterpreterState_Delete(arg1: *mut PyInterpreterState);
 
-    #[cfg(all(Py_3_9, not(PyPy)))]
+    #[cfg(all(any(not(Py_LIMITED_API), Py_3_9), not(PyPy)))]
     #[cfg_attr(docsrs, doc(all(Py_3_9, not(PyPy))))]
-    pub fn PyInterpreterState_Get() -> *mut PyInterpreterState;
+    pub fn PyInterpreterState_Get(arg1: *mut PyInterpreterState) -> *mut PyInterpreterState;
```

Comment:
This change looks incorrect, the corresponding C definition takes no arguments.

Reply:
Thanks for catching this.